### ISSUE TITLE
Create Ivy build

### ISSUE THREAD

**Issue by btxtiger** [VadimDez/ngx-order-pipe#154]
```sh
- ngx-order-pipe [es2015/esm2015] (git+https://github.com/VadimDez/ngx-order-pipe.git)
Encourage the library authors to publish an Ivy distribution.
```

**Comment by nerumo** [comment]
@VadimDez can you have a look at the PR?

**Comment by btxtiger** [comment]
![image](https://github.com/VadimDez/ngx-order-pipe/assets/15804690/1a43fd91-b0ce-4284-b3ec-10195445324c)

Now its breaking with Angular 16. If it's not yet, just delete your `node_modules` folder and try to compile again.

**Comment by stormbreakR1** [comment]
Anyone who have faced this same issue and found any replacement, plz do share as authors of this plugin is not publishing any update for ivy.

**Comment by btxtiger** [comment]
@stormbreakR1 is super simple to create a custom pipe and use lodash orderBy oder lodash sortBy for sorting. This way you could also keep the type if you implement it as generic

**Comment by stormbreakR1** [comment]
@btxtiger thanks

**Comment by reda-alaoui** [comment]
🙏

**Comment by hardcorelite** [comment]
npm i ngx-pipes

**Comment by nerumo** [comment]
still a problem

**Comment by btxtiger** [comment]
@nerumo 
`npm i -S lodash-es`

```ts
import { Pipe, PipeTransform } from '@angular/core';
import { orderBy as _orderBy } from 'lodash-es';

@Pipe({
   name: 'orderBy',
   standalone: true,
})
export class NgxOrderPipe<T> implements PipeTransform {
   transform(data: T[], orderBy: string, direction: 'asc' | 'desc' = 'asc'): T[] {
      return _orderBy(data, orderBy, direction);
   }
}
```

**Comment by nerumo** [comment]
> @nerumo 
> `npm i -S lodash-es`
> 
> ```ts
> import { Pipe, PipeTransform } from '@angular/core';
> import { orderBy as _orderBy } from 'lodash-es';
> 
> @Pipe({
>    name: 'orderBy',
>    standalone: true,
> })
> export class NgxOrderPipe<T> implements PipeTransform {
>    transform(data: T[], orderBy: string, direction: 'asc' | 'desc' = 'asc'): T[] {
>       return _orderBy(data, orderBy, direction);
>    }
> }
> ```

If this is the course of action you suggest, then this repository should be archived

**Comment by btxtiger** [comment]
@nerumo This is a quickfix for most usecases.
I would rather takeover this repo, since the original author is not answering anymore.

**Comment by VadimDez** [comment]
Fixed by version `3.0.0`
